flow/raster_cache.h

```diff
@@ -78,20 +78,39 @@ struct RasterCacheMetrics {
   size_t in_use_bytes = 0;
 
   /**
-   * The total cache entries that had images during this frame whether
-   * they were used in the frame or held memory during the frame and then
-   * were evicted after it ended.
+   * The total cache entries that had images during this frame.
    */
-  size_t total_count() const { return in_use_count + eviction_count; }
+  size_t total_count() const { return in_use_count; }
 
   /**
-   * The size of all of the cached images during this frame whether
-   * they were used in the frame or held memory during the frame and then
-   * were evicted after it ended.
+   * The size of all of the cached images during this frame.
    */
-  size_t total_bytes() const { return in_use_bytes + eviction_bytes; }
+  size_t total_bytes() const { return in_use_bytes; }
 };
 
+/**
+ * RasterCache is used to cache rasterized layers or display lists to improve
+ * performance.
+ *
+ * Life cycle of RasterCache methods:
+ * - Preroll stage
+ *   - RasterCacheItem::PrerollSetup
+ *       At the start of each layer's preroll, add cache items to
+ * `PrerollContext::raster_cached_entries`.
+ *   - RasterCacheItem::PrerollFinalize
+ *       At the end of each layer's preroll, may mark cache entris as
+ * encountered by the current frame.
+ * - Paint stage
+ *   - RasterCache::EvictUnusedCacheEntries
+ *       Evit cached images that are no longer used.
+ *   - LayerTree::TryToPrepareRasterCache
+ *       Create cache image for each cache entry if it does not exist.
+ *   - LayerTree::Paint
```

Comment:
- LayerTree::Paint - for each layer in the tree:

Reply:
Done.

Reply:
Done.